packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -129,7 +130,7 @@ function resolveRequest (acc, resolveReducer) {
 
       // this is useful in the case the error itself is not logged by the
       // implementation
-      console.info(redactedError.toString(), message)
+      if (!omitLog) console.info(redactedError.toString(), message)
```

Comment:
in order for this to be useful, the user would need a way of toggling showErr to be true/false. however, I think we can do what @paulmolluzzo suggested in the issue, and just remove the console.info entirely - the message is included in the error, so the users can choose to log that info if they choose